apiserver/charms_test.go

```diff
@@ -470,7 +470,8 @@ func (s *charmsSuite) TestGetUsesCache(c *gc.C) {
 	uri := s.charmsURI(c, "?url=local:trusty/django-42&file=utils.js")
 	resp, err := s.authRequest(c, "GET", uri, "", nil)
 	c.Assert(err, jc.ErrorIsNil)
-	s.assertGetFileResponse(c, resp, contents, "application/javascript")
+	// Bug 1494726 - Centos generates a content-type of "application/x-javascript"
+	s.assertGetFileResponse(c, resp, contents, `application/(x-javascript|javascript)`)
```

Comment:
if we can just change the test without changing any code... then the code doesn't actually care what the content-type is.  So why does the test care?